packages/design-system/.prettierrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Are we also trying to have cross-repo consistency about `prettier`? Does that mean this should go somewhere else (long-term) rather than living in `design-system/`?

Reply:
yeah good point, others have asked me about this also and I keep neglecting it. I looked into it and it turns out you can share and extend them in basically the same way you can with eslint configurations. I'll open an issue for dealing with this!